src/alogamous/flag_duplicate_log_messages.py

```diff
@@ -0,0 +1,20 @@
+from alogamous import analyzer
+
+
+class FlagDuplicateLogMessages(analyzer.Analyzer):
+    def __init__(self):
+        self.logMessages = []
+        self.duplicateMessages = set()
+
+    def read_log_line(self, line):
+        message = line.split("-")[-1].strip()
+        self.logMessages.append(message)
+
+        for message in self.logMessages:
+            if self.logMessages.count(message) > 1:
```

Comment:
One problem with this approach is that, since `logMessages` is a list, as log files get bigger this will get slower and slower. If you made `logMessages` a set and did something like `message in self.logMessages` instead of `self.logMessages.count(message) > 1`, it would run a lot faster with large logs.

Reply:
You can see the [time complexity](https://en.wikipedia.org/wiki/Time_complexity) for these operations [here](https://wiki.python.org/moin/TimeComplexity)

If you take Computer Science in college, you'll probably learn about this in your Data Structures class.

Reply:
I added the changes, it should be good now!